templates/policy-compliance-agent/backend/services/embedding_manager.py

```
@@ -0,0 +1,202 @@
+    def _initialize_text_splitter(self):
+        """Initialize GraphBit text splitter with policy-optimized configuration."""
+        try:
+            # Configure recursive splitter for policy content
+            self.text_splitter = SentenceSplitter(chunk_size=1000, chunk_overlap=200)
+
+            logger.info("GraphBit text splitter initialized for policy documents")
+
+        except ImportError as e:
+            logger.warning(f"GraphBit not available for text splitting: {e}")
+            self.text_splitter = None
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)